src/deadline/blender_adaptor/BlenderClient/render_handlers/default_blender_handler.py

```diff
@@ -115,6 +115,11 @@ def start_render(self, data: dict) -> None:
             f"{self.output_dir}/{self.view_layer_name}_{camera}_{self.output_file_name}"
         )
         _logger.debug(f"Set output file path to {bpy.context.scene.render.filepath}")
+
+        # Add the output format
+        bpy.context.scene.render.image_settings.file_format = self.format
```

Comment:
nit: The adaptor schema isn't an enum so I'd be curious what happens if someone isn't using the submitter but is using the adaptor and their own template that isn't a "valid" file format.

Reply:
I'll need to make some changes anyway so I can test this out + add validation on the adaptor's end if necessary.